pycheribuild/projects/disk_image.py

```diff
@@ -483,6 +483,28 @@ def run_mkimg(self, cmd: list, **kwargs):
             self.fatal("Missing mkimg command! Should be found in FreeBSD build dir (or set $MKIMG_CMD)")
         self.run_cmd([self.mkimg_cmd] + cmd, **kwargs)
 
+    @property
+    def include_efi_partition(self):
+        if self.crosscompile_target.is_mips(include_purecap=True):
+            return False
+        # TODO: Make this unconditional once all branches support EFI
+        if self.crosscompile_target.is_riscv(include_purecap=True):
```

Comment:
Why is this only for RISC-V? make_aarch64_disk_image was using an EFI partition too?

Reply:
I screwed up the original patch, should be fixed already (defaults to True, with the MIPS early bail-out)